metadata/utils.go

```diff
@@ -113,6 +113,21 @@ func IsRawPayload(props map[string]string) (bool, error) {
 
 	return false, nil
 }
+func TryGetMessageId(props map[string]string) (string, bool) {
```

Comment:
This is not the right place for your RabbitMQ component specific shared code.

Instead of `metadata/utils.go`, create a `rabbitmq` folder here: https://github.com/dapr/components-contrib/tree/main/common/component

Then import from there. Please add a unit test for these helper methods.